evap/staff/views.py

```diff
@@ -1500,7 +1500,11 @@ def evaluation_textanswers(request, semester_id, evaluation_id):
 
 @reviewer_required
 def evaluation_textanswers_skip(request):
-    evaluation_id = request.POST["evaluation_id"]
+    try:
+        evaluation_id = request.POST["evaluation_id"]
+    except KeyError:
+        return unexpected_bad_request()
+
     evaluation = get_object_or_404(Evaluation, id=evaluation_id)
```

Comment:
Can't we just do `request.POST.get("evaluation_id")` and have the 404 returned by `get_object_or_404`? Edit: Ah, do we want to distinguish between 400 and 404? If so, forget what I said :D

Reply:
Yes, I explicitly wanted to keep the logging of any request that doesn't have a `evaluation_id` in its data, because I'd consider it likely to indicate a bug at another place.

At the same time, the same would be true for the 404. Haven't thought about that.